src/core/thread/mle.cpp

```diff
@@ -637,6 +641,17 @@ void Mle::SetAssignLinkQuality(const Mac::ExtAddress aMacAddr, uint8_t aLinkQual
     }
 }
 
+uint8_t Mle::GetRouterSelectionJitter(void) const
+{
+    return mSetRouterJitter ? mRouterJitter : mRouterSelectionJitterTimeout;
+}
+
+void Mle::SetRouterSelectionJitter(uint8_t aRouterJitter)
+{
+    mSetRouterJitter = true;
```

Comment:
Yes, that also makes sense. I think both `HandleChildIdResponse` and `HandleAdvertisement` should use the same constant `kRouterSelectionJitter`(120s). Did not find one case requires a different value. Let me update this.